src/test_utils.rs

```diff
@@ -42,3 +42,10 @@ impl AsRef<Path> for TempFixture {
         self.path()
     }
 }
+
+#[macro_export]
+macro_rules! fixture {
+    ($name:expr) => {
+        std::path::Path::new(env!("FIXTURES_DIR")).join($name)
```

Comment:
Should we just convert this into a function? Seems silly now to make it a macro...